content/en/dora_metrics/deployments/apm.md

```diff
@@ -0,0 +1,129 @@
+- link: "/continuous_integration/dora_metrics/setup/incidents"
+  tag: "Documentation"
+  text: "Learn about sending incident events"
+- link: "/tracing/service_catalog"
+  tag: "Documentation"
+  text: "Learn about the Service Catalog"
+- link: "https://github.com/DataDog/datadog-ci"
+  tag: "Source Code"
+  text: "Learn about the datadog-ci CLI tool"
+- link: "/continuous_delivery/deployments"
+  tag: "Documentation"
+  text: "Learn about Deployment Visibility"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">DORA Metrics is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+{{< callout url="https://forms.gle/Eqq6uXfGjYxmqpjDA" header="false" >}}
+The DORA Metrics private beta is closed. Fill out the form below to be added to the waitlist.
+{{< /callout >}}
+
+## Overview
+APM [Deployment Tracking][2] can be configured as a data source for deployments in DORA Metrics.
+
+For service deployments tracked by APM to contribute to DORA Metrics, the following requirements must be met:
+- Your service has [metadata][1] defined in the Service Catalog.
+- Your service has [unified service tagging][3] enabled. Deployments are identified via the `version` tag.
+
+## Change lead time
+For service deployments tracked by APM to contribute to Change Lead Time, the following requirement must be met:
+- Your application telemetry is tagged with git information. You can enable this [in APM][4] or see [documentation][5].
+- Your repository metadata is synchronized to Datadog via the GitHub integration.
+
+For deployments identified via Deployment Tracking, change lead time is computed from time of first commit creation to when that commit is first seen in a new version.
+
+### Breakdown metrics
+Datadog breaks down change lead time into the following metrics, which represent the different stages from commit creation to deployment.
+
+To compute these metrics, there must be a PR associated with a commit, if any. A commit is associated with a PR if the commit is first introduced to the target branch when merging that PR.
+
+If a commit does not have an associated PR, only Time to Deploy and Deploy Time are available.
+
+- `dora.time_to_pr_ready`: Time from when the commit is created until the PR is ready for review. This metric is only available for commits that were made before the PR is ready for review.
+- `dora.review_time`: Time from when the PR is marked ready for review until it receives the last approval. This metric is only available for commits that were made before the PR is approved.
+- `dora.merge_time`: Time from the last approval until the PR is merged.
+- `dora.time_to_deploy`: Time from PR merge to start of deployment. If a commit does not have an associated PR, this metric is calculated as the time from commit creation to start of deployment.
+
+### Synchronize repository metadata to Datadog
```

Comment:
Should we have this and the next section on a common page? It's a duplicate of content/en/dora_metrics/deployments/_index.md, right?

Reply:
Yeah, I suppose there aren't enough differences right now in how the different data sources handle repo metadata. Will make this change, thanks!